### PR TITLE
Payara 6 `javax.*` -> `jakarta.*`

### PR DESCRIPTION
**Pull request by JamesHillyard**
Changes `javax` to `jakarta` namespace. 

javax.cache hasn't been changes as this is from Hazelcast 5.1.1 and still uses javax.cache.

`javax.net.ssl.trustStore` and similar JVM properties also weren't changed.

### CODE REVIEW

File: docs/modules/ROOT/pages/Technical Documentation/MicroProfile/Rest Client.adoc [hunk 1]
```diff
@@ -70,7 +70,7 @@ On a per-client basis, or via Microprofile Configuration properties, the followi
 * Keystore for setting up mutual SSL trust
 * Hostname verifier, useful for suppressing validity checks for whitelisted host names
 
-Programmatically it is also possible to provide an implementation of the `javax.net.ssl.SSLContext` class to have major control over all aspects of this secure communication channel.
+Programmatically it is also possible to provide an implementation of the `jakarta.net.ssl.SSLContext` class to have major control over all aspects of this secure communication channel.
```
Comment (Pandrex247) [inline docs/modules/ROOT/pages/Technical Documentation/MicroProfile/Rest Client.adoc:73]: Don't think this one has changed.

File: docs/modules/ROOT/pages/Technical Documentation/Payara Micro Documentation/Payara Micro Configuration and Management/Database Management/Log JDBC Calls.adoc [hunk 2]
```diff
@@ -51,7 +51,7 @@ or in the definition of a `jdbc-connection-pool` in a __payara-resources.xml__ f
 [source, xml]
 ----
 <jdbc-connection-pool name="examplePool"
-        res-type="javax.sql.DataSource"
```
Comment (Pandrex247) [inline docs/modules/ROOT/pages/Technical Documentation/Payara Micro Documentation/Payara Micro Configuration and Management/Database Management/Log JDBC Calls.adoc:54]: Don't think this one has changed.

File: docs/modules/ROOT/pages/Technical Documentation/Payara Micro Documentation/Payara Micro Configuration and Management/Database Management/Log JDBC Calls.adoc [hunk 3]
```diff
@@ -63,7 +63,7 @@ or in the definition of a `jdbc-connection-pool` in a __payara-resources.xml__ f
 This can also be done in the creation of a JDBC Connection Pool with the `--logjdbccalls` option.
 [source, shell]
 ----
-asadmin create jdbc-connection-pool --datasourceclassname org.h2.jdbcx.JdbcDataSource --restype javax.sql.XADataSource --logjdbccalls=true examplePool
+asadmin create jdbc-connection-pool --datasourceclassname org.h2.jdbcx.JdbcDataSource --restype jakarta.sql.XADataSource --logjdbccalls=true examplePool
```
Comment (Pandrex247) [inline docs/modules/ROOT/pages/Technical Documentation/Payara Micro Documentation/Payara Micro Configuration and Management/Database Management/Log JDBC Calls.adoc:66]: Don't think this one has changed.

File: docs/modules/ROOT/pages/Technical Documentation/Payara Server Documentation/Jakarta EE API/Enterprise Java Beans (EJB)/Overview.adoc [hunk 4]
```diff
@@ -340,11 +340,11 @@ public class Bean implements BeanRemote, Serializable {
 +
 [source, java]
 ----
-import static javax.naming.Context.INITIAL_CONTEXT_FACTORY;
-import static javax.naming.Context.PROVIDER_URL;
+import static jakarta.naming.Context.INITIAL_CONTEXT_FACTORY;
```
Comment (Pandrex247) [inline docs/modules/ROOT/pages/Technical Documentation/Payara Server Documentation/Jakarta EE API/Enterprise Java Beans (EJB)/Overview.adoc:343]: Don't think this one has changed.

File: docs/modules/ROOT/pages/Technical Documentation/Payara Server Documentation/Jakarta EE API/Enterprise Java Beans (EJB)/Overview.adoc [hunk 5]
```diff
@@ -340,11 +340,11 @@ public class Bean implements BeanRemote, Serializable {
...
+import static jakarta.naming.Context.PROVIDER_URL;
```
Comment (Pandrex247) [inline docs/modules/ROOT/pages/Technical Documentation/Payara Server Documentation/Jakarta EE API/Enterprise Java Beans (EJB)/Overview.adoc:344]: Don't think this one has changed.

File: docs/modules/ROOT/pages/Technical Documentation/Payara Server Documentation/Jakarta EE API/Enterprise Java Beans (EJB)/Overview.adoc [hunk 6]
```diff
@@ -340,11 +340,11 @@ public class Bean implements BeanRemote, Serializable {
...
 import java.util.Properties;
-import javax.naming.Context;
-import javax.naming.InitialContext;
+import jakarta.naming.Context;
```
Comment (Pandrex247) [inline docs/modules/ROOT/pages/Technical Documentation/Payara Server Documentation/Jakarta EE API/Enterprise Java Beans (EJB)/Overview.adoc:346]: Don't think this one has changed.

File: docs/modules/ROOT/pages/Technical Documentation/Payara Server Documentation/Jakarta EE API/Enterprise Java Beans (EJB)/Overview.adoc [hunk 7]
```diff
@@ -340,11 +340,11 @@ public class Bean implements BeanRemote, Serializable {
...
+import jakarta.naming.InitialContext;
```
Comment (Pandrex247) [inline docs/modules/ROOT/pages/Technical Documentation/Payara Server Documentation/Jakarta EE API/Enterprise Java Beans (EJB)/Overview.adoc:347]: Don't think this one has changed.

File: docs/modules/ROOT/pages/Technical Documentation/Payara Server Documentation/Jakarta EE API/Enterprise Java Beans (EJB)/Overview.adoc [hunk 8]
```diff
@@ -397,20 +397,20 @@ Then, the corresponding HTTP endpoint to use would be `https://localhost:8181/ej
 +
 Then the credentials required to correctly authenticate the user for the call have to be specified in the JNDI context with the following properties:
 +
-* `javax.naming.Context.SECURITY_PRINCIPAL` for the username
-* `javax.naming.Context.SECURITY_CREDENTIALS` for the password
+* `jakarta.naming.Context.SECURITY_PRINCIPAL` for the username
```
Comment (Pandrex247) [inline docs/modules/ROOT/pages/Technical Documentation/Payara Server Documentation/Jakarta EE API/Enterprise Java Beans (EJB)/Overview.adoc:400]: Don't think this one has changed.

File: docs/modules/ROOT/pages/Technical Documentation/Payara Server Documentation/Jakarta EE API/Enterprise Java Beans (EJB)/Overview.adoc [hunk 9]
```diff
@@ -397,20 +397,20 @@ Then, the corresponding HTTP endpoint to use would be `https://localhost:8181/ej
...
+* `jakarta.naming.Context.SECURITY_CREDENTIALS` for the password
```
Comment (Pandrex247) [inline docs/modules/ROOT/pages/Technical Documentation/Payara Server Documentation/Jakarta EE API/Enterprise Java Beans (EJB)/Overview.adoc:401]: Don't think this one has changed.

File: docs/modules/ROOT/pages/Technical Documentation/Payara Server Documentation/Jakarta EE API/Enterprise Java Beans (EJB)/Tracing Remote EJBs.adoc [hunk 10]
```diff
@@ -12,10 +12,10 @@ No additional properties need to be specified when performing the initial contex
...
 [source, java]
 ----
-import javax.naming.Context;
-import javax.naming.InitialContext;
-import javax.naming.NamingException;
-import java.util.Properties;
+import jakarta.naming.Context;
+import jakarta.naming.InitialContext;
+import jakarta.naming.NamingException;
+import jakarta.util.Properties;
```
Comment (Pandrex247) [inline docs/modules/ROOT/pages/Technical Documentation/Payara Server Documentation/Jakarta EE API/Enterprise Java Beans (EJB)/Tracing Remote EJBs.adoc:18]: Don't think these have changed.

File: docs/modules/ROOT/pages/Technical Documentation/Payara Server Documentation/Jakarta EE API/Enterprise Java Beans (EJB)/Overview.adoc [hunk 11]
```diff
@@ -397,20 +397,20 @@ Then, the corresponding HTTP endpoint to use would be `https://localhost:8181/ej
...
-import static javax.naming.Context.SECURITY_PRINCIPAL;
+import static jakarta.naming.Context.INITIAL_CONTEXT_FACTORY;
+import static jakarta.naming.Context.PROVIDER_URL;
+import static jakarta.naming.Context.SECURITY_CREDENTIALS;
+import static jakarta.naming.Context.SECURITY_PRINCIPAL;
 import java.util.Properties;
-import javax.naming.Context;
-import javax.naming.InitialContext;
+import jakarta.naming.Context;
+import jakarta.naming.InitialContext;
```
Comment (Pandrex247) [inline docs/modules/ROOT/pages/Technical Documentation/Payara Server Documentation/Jakarta EE API/Enterprise Java Beans (EJB)/Overview.adoc:413]: Don't think these have changed.

File: docs/modules/ROOT/pages/Technical Documentation/Payara Server Documentation/Jakarta EE API/Enterprise Java Beans (EJB)/Overview.adoc [hunk 12]
```diff
@@ -450,10 +450,10 @@ Under the covers the remote EJB proxy uses a JAX-RS (Jersey) REST client builder
 | Instance of `java.security.KeyStore` or a `String` representing its fully qualified classname.
 |`fish.payara.sslContext`
 | The SSL context that will be used by the proxy for creating secured connections to the Payara remote server. This context *must* be fully initialized, including the trust and key managers. Should not be used in conjunction with the `fish.payara.keyStore` and/or `fish.payara.trustStore` properties.
-| Instance of `javax.net.ssl.SSLContext` or a `String` representing its fully qualified classname.
+| Instance of `jakarta.net.ssl.SSLContext` or a `String` representing its fully qualified classname.
```
Comment (Pandrex247) [inline docs/modules/ROOT/pages/Technical Documentation/Payara Server Documentation/Jakarta EE API/Enterprise Java Beans (EJB)/Overview.adoc:453]: Don't think this one has changed.

File: docs/modules/ROOT/pages/Technical Documentation/Payara Server Documentation/Jakarta EE API/Enterprise Java Beans (EJB)/Overview.adoc [hunk 13]
```diff
@@ -450,10 +450,10 @@ Under the covers the remote EJB proxy uses a JAX-RS (Jersey) REST client builder
...
 |`fish.payara.hostnameVerifier`
 | The hostname verifier to be used by the proxy to verify the endpoint's hostname against the identification information of it.
-| Instance of a `javax.net.ssl.HostnameVerifier` or a `String` representing its fully qualified classname.
+| Instance of a `jakarta.net.ssl.HostnameVerifier` or a `String` representing its fully qualified classname.
```
Comment (Pandrex247) [inline docs/modules/ROOT/pages/Technical Documentation/Payara Server Documentation/Jakarta EE API/Enterprise Java Beans (EJB)/Overview.adoc:456]: Don't think this one has changed.

File: docs/modules/ROOT/pages/Technical Documentation/Payara Server Documentation/Jakarta EE API/Enterprise Java Beans (EJB)/Overview.adoc [hunk 14]
```diff
@@ -488,7 +488,7 @@ The constants are also exposed as static attributes of the `fish.payara.ejb.rest
 [[system-properties-fallbacks]]
 ==== System Properties Fallbacks
 
-The JNDI customization options listed above as well the environment variables defined in `javax.naming.Context` (except `APPLET`) can be set by setting a system property of the same name that will act as a fallback.
+The JNDI customization options listed above as well the environment variables defined in `jakarta.naming.Context` (except `APPLET`) can be set by setting a system property of the same name that will act as a fallback.
```
Comment (Pandrex247) [inline docs/modules/ROOT/pages/Technical Documentation/Payara Server Documentation/Jakarta EE API/Enterprise Java Beans (EJB)/Overview.adoc:491]: Don't think this one has changed.

File: docs/modules/ROOT/pages/Technical Documentation/Payara Server Documentation/Jakarta EE API/Enterprise Java Beans (EJB)/Tracing Remote EJBs.adoc [hunk 15]
```diff
@@ -44,10 +44,10 @@ NOTE: The built-in Payara Request Tracing service does not support tracing of Ja
...
 import io.opentracing.util.GlobalTracer;
 
-import javax.naming.Context;
-import javax.naming.InitialContext;
-import javax.naming.NamingException;
-import java.util.Properties;
+import jakarta.naming.Context;
+import jakarta.naming.InitialContext;
+import jakarta.naming.NamingException;
+import jakarta.util.Properties;
```
Comment (Pandrex247) [inline docs/modules/ROOT/pages/Technical Documentation/Payara Server Documentation/Jakarta EE API/Enterprise Java Beans (EJB)/Tracing Remote EJBs.adoc:50]: Don't think these have changed.

File: docs/modules/ROOT/pages/Technical Documentation/Payara Server Documentation/Jakarta EE API/Enterprise Java Beans (EJB)/Tracing Remote EJBs.adoc [hunk 16]
```diff
@@ -84,10 +84,10 @@ import io.opentracing.Span;
...
```
Comment (Pandrex247) [inline docs/modules/ROOT/pages/Technical Documentation/Payara Server Documentation/Jakarta EE API/Enterprise Java Beans (EJB)/Tracing Remote EJBs.adoc:90]: Don't think these have changed.

File: docs/modules/ROOT/pages/Technical Documentation/Payara Server Documentation/Server Configuration And Management/Domain Data Grid And Hazelcast/Datagrid in Applications.adoc [hunk 17]
```diff
@@ -12,8 +12,8 @@ By default, the JNDI name of the hazelcast instance is `payara/Hazelcast`, thoug
 [source, java]
 ----
 import com.hazelcast.core.HazelcastInstance;
-import javax.naming.Context;
-import javax.naming.InitialContext;
+import jakarta.naming.Context;
+import jakarta.naming.InitialContext;
```
Comment (Pandrex247) [inline docs/modules/ROOT/pages/Technical Documentation/Payara Server Documentation/Server Configuration And Management/Domain Data Grid And Hazelcast/Datagrid in Applications.adoc:16]: Don't think these have changed.

File: docs/modules/ROOT/pages/Technical Documentation/Payara Server Documentation/Server Configuration And Management/JDBC Resource Management/JDBC.adoc [hunk 18]
```diff
@@ -27,7 +27,7 @@ Click the _New_ button and enter the following values:
 |=======================================================================
 |Setting Name |Value
 |Pool Name |test-pool
-|Resource Type |`javax.sql.DataSource`
+|Resource Type |`jakarta.sql.DataSource`
```
Comment (Pandrex247) [inline docs/modules/ROOT/pages/Technical Documentation/Payara Server Documentation/Server Configuration And Management/JDBC Resource Management/JDBC.adoc:30]: Don't think this has changed.

File: docs/modules/ROOT/pages/Technical Documentation/Payara Server Documentation/Server Configuration And Management/JDBC Resource Management/JDBC.adoc [hunk 19]
```diff
@@ -59,7 +59,7 @@ You can also create a JDBC connection pool using the `asadmin` command line tool
 
 [source, shell]
 ----
-asadmin> create-jdbc-connection-pool --datasourceclassname com.mysql.jdbc.jdbc2.optional.MysqlDataSource --restype javax.sql.DataSource --property user=root:password=test:DatabaseName=test:ServerName=localhost:port=3306 test-pool
+asadmin> create-jdbc-connection-pool --datasourceclassname com.mysql.jdbc.jdbc2.optional.MysqlDataSource --restype jakarta.sql.DataSource --property user=root:password=test:DatabaseName=test:ServerName=localhost:port=3306 test-pool
```
Comment (Pandrex247) [inline docs/modules/ROOT/pages/Technical Documentation/Payara Server Documentation/Server Configuration And Management/JDBC Resource Management/JDBC.adoc:62]: Don't think this has changed.

File: docs/modules/ROOT/pages/Technical Documentation/Payara Server Documentation/Server Configuration And Management/JDBC Resource Management/JDBC.adoc [hunk 20]
```diff
@@ -301,7 +301,7 @@ or in the definition of a `jdbc-connection-pool` in a __payara-resources.xml__ f
 [source, xml]
 ----
 <jdbc-connection-pool name="examplePool"
-        res-type="javax.sql.DataSource"
+        res-type="jakarta.sql.DataSource"
```
Comment (Pandrex247) [inline docs/modules/ROOT/pages/Technical Documentation/Payara Server Documentation/Server Configuration And Management/JDBC Resource Management/JDBC.adoc:304]: Don't think this has changed.

File: docs/modules/ROOT/pages/Technical Documentation/Payara Server Documentation/Server Configuration And Management/JDBC Resource Management/JDBC.adoc [hunk 21]
```diff
@@ -314,7 +314,7 @@ This can also be done in the creation of a JDBC Connection Pool with the `--logj
 
 [source, shell]
 ----
-asadmin create jdbc-connection-pool --datasourceclassname org.h2.jdbcx.JdbcDataSource --restype javax.sql.XADataSource --logjdbccalls=true examplePool
+asadmin create jdbc-connection-pool --datasourceclassname org.h2.jdbcx.JdbcDataSource --restype jakarta.sql.XADataSource --logjdbccalls=true examplePool
```
Comment (Pandrex247) [inline docs/modules/ROOT/pages/Technical Documentation/Payara Server Documentation/Server Configuration And Management/JDBC Resource Management/JDBC.adoc:317]: Don't think this has changed.

File: docs/modules/ROOT/pages/Technical Documentation/Payara Server Documentation/Server Configuration And Management/JDBC Resource Management/JDBC.adoc [hunk 22]
```diff
@@ -240,7 +240,7 @@ With *Log JDBC Calls* configured on the connection call each call into the conne
 
 [source, log]
 ----
-[#|2016-02-04T18:51:01.467+0000|FINE|Payara 6.2022.1|javax.enterprise.resource.sqltrace.com.sun.gjc.util|_ThreadID=35;_ThreadName=http-listener-1(5);_TimeMillis=1454611861467;_LevelValue=500;ClassName=com.sun.gjc.util.SQLTraceLogger;MethodName=sqlTrace;|
+[#|2016-02-04T18:51:01.467+0000|FINE|Payara 6.2022.1|jakarta.enterprise.resource.sqltrace.com.sun.gjc.util|_ThreadID=35;_ThreadName=http-listener-1(5);_TimeMillis=1454611861467;_LevelValue=500;ClassName=com.sun.gjc.util.SQLTraceLogger;MethodName=sqlTrace;|
```
Comment (Pandrex247) [inline docs/modules/ROOT/pages/Technical Documentation/Payara Server Documentation/Server Configuration And Management/JDBC Resource Management/JDBC.adoc:243]: Really not sure on this one...
Not sure where it's pulling this package name from - it looks like two concatenated together.

`com.sun.gjc.util` is from our code base, not sure about `*.enterprise.resource.sqltrace`.

Our logging.properties still refers to it as _javax_, but I don't know if that's correct...

**Review by Pandrex247 — CHANGES_REQUESTED** [review]

**Review by Pandrex247 — CHANGES_REQUESTED** [review]

**Review by Pandrex247 — CHANGES_REQUESTED** [review]
* Link to `RolesAllowed` documentation in `Roles Permitted` page probably needs updating.
* Link to `IdentityStore` in `Public API` probably needs updating
* Link to `IdentityStore` in `Security Connectors` probably needs updating
* Link to `HttpAuthenticationMechanism` in `Security Connectors` probably needs updating

**Comment by JamesHillyard** [comment]
Updated those links, also updated the JavaSE 8 links to the 11 equivalents

**Review by Pandrex247 — APPROVED** [review]

**Pull request merged**